docs/modules/ROOT/pages/governance/governor.adoc

```diff
@@ -0,0 +1,450 @@
+to implement the {votes-component} extension. This extension will keep track of historical balances so that voting power
+is retrieved from past snapshots rather than current balance, which is an important protection that prevents double voting.
+
+If your project already has a live token that does not include Votes and is not upgradeable, you can wrap it in a
+governance token by using a wrapper. This will allow token holders to participate in governance by wrapping their tokens 1-to-1.
+
+NOTE: The library currently does not include a wrapper for tokens, but it will be added in a future release.
+
+NOTE: Currently, the clock mode is fixed to block timestamps, since the Votes component uses the block timestamp to track
+checkpoints. We plan to add support for more flexible clock modes in Votes in a future release, allowing to use, for example,
+block numbers instead.
+
+=== Governor
+
+We will initially build a Governor without a timelock. The core logic is given by the {governor-component}, but we
+still need to choose:
+
+1) how voting power is determined,
+
+2) how many votes are needed for quorum,
+
+3) what options people have when casting a vote and how those votes are counted, and
+
+4) the execution mechanism that should be used.
+
+Each of these aspects is customizable by writing your own extensions,
+or more easily choosing one from the library.
+
+:GovernorVotes: xref:api/governance.adoc#GovernorVotesComponent[GovernorVotes]
+:GovernorVotesQuorumFraction: xref:api/governance.adoc#GovernorVotesQuorumFractionComponent[GovernorVotesQuorumFraction]
+:GovernorCountingSimple: xref:api/governance.adoc#GovernorCountingSimpleComponent[GovernorCountingSimple]
+:GovernorCoreExecution: xref:api/governance.adoc#GovernorCoreExecutionComponent[GovernorCoreExecution]
+:GovernorSettings: xref:api/governance.adoc#GovernorSettingsComponent[GovernorSettings]
+:GovernorTimelockExecution: xref:api/governance.adoc#GovernorTimelockExecutionComponent[GovernorTimelockExecution]
+:GovernorSettingsTrait: xref:api/governance.adoc#GovernorComponent[GovernorSettingsTrait]
+
+**For 1)** we will use the {GovernorVotes} extension, which hooks to an {ivotes} instance to determine the voting power
+of an account based on the token balance they hold when a proposal becomes active.
+This module requires the address of the token to be passed as an argument to the initializer.
+
+**For 2)** we will use {GovernorVotesQuorumFraction}. This works together with the {ivotes} instance to define the quorum as a
+percentage of the total supply at the block when a proposal’s voting power is retrieved. This requires an initializer
+parameter to set the percentage besides the votes token address. Most Governors nowadays use 4%, so we will initialize
+the module with parameter 40 since the quorum denominator is 1000 for precision (this indicates the percentage,
+resulting in 4%).
```

Comment:
This part could be rephrased to improve clarity:

"Most Governors nowadays use a quorum of 4%. Since the quorum denominator is 1000 for precision, we initialize the module with a numerator of 40, resulting in a 4% quorum (40/1000 = 0.04 or 4%)"

Reply:
Better indeed.